COMMON: chart-testing-action fails

```
Run helm/chart-testing-action@v2.4.0
Run sigstore/cosign-installer@c[36](https://github.com/intershop/helm-charts/actions/runs/6769780166/job/18396872595?pr=449#step:3:42)67d99424e7e6047999fb6246c0da843953c65
Run #!/bin/bash
INFO: Downloading bootstrap version 'v2.0.0' of cosign to verify version to be installed...
      https://storage.googleapis.com/cosign-releases/v2.0.0/cosign-linux-amd64
ERROR: Unable to validate cosign version: 'v2.0.0'
Error: Process completed with exit code 1.
```

--- a/.github/template/base/action.yml
+++ b/.github/template/base/action.yml
@@ -16,7 +16,7 @@ runs:
       python-version: 3.7
 
   - name: Set up chart-testing
-    uses: helm/chart-testing-action@v2.4.0
+    uses: helm/chart-testing-action@v2.6.1
 
   - name: Install Helm Unittest
     shell: bash

